presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -848,6 +881,111 @@ public void testExecute()
     }
 
     @Test
+    public void testExecuteWithInsert()
+            throws Exception
+    {
+        try (Connection connection = createConnection("blackhole", "blackhole")) {
+            try (Statement statement = connection.createStatement()) {
+                assertFalse(statement.execute("INSERT INTO test_table VALUES (1)"));
+
+                assertNull(statement.getResultSet());
+                assertEquals(statement.getUpdateCount(), 1);
+                assertEquals(statement.getLargeUpdateCount(), 1L);
```

Comment:
All of these `getLargeUpdateCount()` checks can use `0` instead of `0L` since they are compared against a primitive long. The `0L` is only needed if the other side is an object (in which case the `0` would incorrectly box to `Integer`).